release/setup.py

```diff
@@ -50,7 +50,9 @@ def finalize_options(self):
             self.install_lib = self.install_platlib
 
 
-REQUIRED_PACKAGES = ['cirq == 0.9.1', 'sympy == 1.5']
+REQUIRED_PACKAGES = [
```

Comment:
I realized that this can be required from the empty virtualenv:

```
google-api-core==1.21.0
google-auth==1.18.0
googleapis-common-protos==1.52.0
grpcio==1.30.0
protobuf==3.12.0
```

Reply:
I tried to install `cirq==0.10.1` last week and failed, and found out this workaround.

Reply:
All these dependencies worked. Although I did need to use protobuf version 3.13.0 instead of 3.12.0 to have everything work.

Reply:
Sounds great. LGTM.